stubs/prettytable/prettytable/prettytable.pyi

```diff
@@ -199,19 +200,25 @@ class PrettyTable:
     def xhtml(self, val: bool) -> None: ...
     @property
     def rows(self) -> list[Incomplete]: ...
+    @property
+    def header_align(self) -> dict[str, str]: ...
+    @header_align.setter
+    def header_align(self, val: Literal["l", "c", "r"] | dict[str, str] | None) -> None: ...
```

Comment:
I'm a little confuse by this, it looks like it maybe only accepts an empty dict?

Reply:
Yep, that's what it looks like to me as well. Think we should go for `dict[Any, Any]` instead? I wasn't really sure what was best here...

Reply:
We could use something like `class _EmptyDict(TypedDict): pass`.